samples/bluetooth/peripheral_power_profiling/prj.conf

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+
+CONFIG_BT=y
+CONFIG_BT_SMP=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_DEVICE_NAME="Nordic_Power_Profiling"
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_USER_PHY_UPDATE=y
+CONFIG_BT_CTLR_PHY_CODED=y
+
+CONFIG_NFC_T2T_NRFXLIB=y
+
+CONFIG_NFC_NDEF=y
+CONFIG_NFC_NDEF_MSG=y
+CONFIG_NFC_NDEF_RECORD=y
+CONFIG_NFC_NDEF_LE_OOB_REC=y
+CONFIG_NFC_NDEF_CH_MSG=y
```

Comment:
Consider removing the NFC OOB logic from this sample since it makes the sample more complex. You can still use the NFCT as a wakeup source without using the NFC library at all. In this case, you can use direct APIs of nrfx.

Reply:
I think it is good to have also OOB pairing in this sample. It shows user that after device is wake up by NFC he can establish connection also

Reply:
I see. I am leaving this comment as unresolved for now in case any other reviewer plans to share their perspective on this topic.